api/swagger.yaml

```diff
@@ -668,6 +722,28 @@ paths:
         default:
           $ref: '#/responses/UnknownError'
 
+  /runtime/scheduleScan/start:
+    put:
```

Comment:
also need a get here.
maybe it's better to do it in the quick scan format
https://github.com/cisco-open/kubei/blob/main/api/swagger.yaml#L755-L783